packages/storybook-html/src/components/button/button.docs.mdx

```diff
@@ -0,0 +1,152 @@
+...
+```
+
+### Expand
+
+Expand version of the button would extend the width of the button to full width of its parent container.
+
+<Canvas withSource="none">
+  <Story id="components-button--expand" />
+</Canvas>
+
+```html
+<button class="uq-button uq-button--expand">Button</button>
+<button class="uq-button uq-button--expand uq-button--outline">Button</button>
+<button class="uq-button uq-button--expand uq-button--text">Button</button>
+
+<button class="uq-button uq-button--secondary uq-button--expand">Button</button>
+<button class="uq-button uq-button--secondary uq-button--expand uq-button--outline">Button</button>
+<button class="uq-button uq-button--secondary uq-button--expand uq-button--text">Button</button>
+
+<button class="uq-button uq-button--secondary uq-button--expand uq-button--outline uq-button--light">Button</button>
+<button class="uq-button uq-button--secondary uq-button--expand uq-button--text uq-button--light">Button</button>
+
+<button class="uq-button uq-button--purple uq-button--expand">Button</button>
+<button class="uq-button uq-button--purple uq-button--expand uq-button--outline">Button</button>
+<button class="uq-button uq-button--purple uq-button--expand uq-button--text">Button</button>
+```
+
+## How and when to use
+
+<table className="uqds-table  uqds-table--card uqds-table--colour">
+  <tbody className="uqds-table__body">
+    <tr className="uqds-table__mobile-heading">
+      <th className="uqds-table__heading">Category</th>
+      <th className="uqds-table__heading">Styles</th>
+      <th className="uqds-table__heading">How/When to use:</th>
+    </tr>
+    <tr className="uqds-table__row">
+      <td className="uqds-table__item" data-title="Category:">Primary</td>
+      <td className="uqds-table__item" data-title="Styles:">Normal, outline, text, expand</td>
+      <td className="uqds-table__item" data-title="How/When to use:">Used for most important call to action button on the webpage</td>
+    </tr>
+    <tr className="uqds-table__row">
+      <td className="uqds-table__item" data-title="Category:">Secondary</td>
+      <td className="uqds-table__item" data-title="Styles:">Normal, outline, text, expand</td>
+      <td className="uqds-table__item" data-title="How/When to use:">Used for important actions like save etc.</td>
+    </tr>
+    <tr className="uqds-table__row">
+      <td className="uqds-table__item" data-title="Category:">Secondary light</td>
+      <td className="uqds-table__item" data-title="Styles:">Normal, outline, text, expand</td>
+      <td className="uqds-table__item" data-title="How/When to use:">Used on dark backgrounds</td>
+    </tr>
+    <tr className="uqds-table__row">
+      <td className="uqds-table__item" data-title="Category:">Purple</td>
+      <td className="uqds-table__item" data-title="Styles:">Normal, outline, text, expand</td>
+      <td className="uqds-table__item" data-title="How/When to use:">Used in UQ events and stories sections</td>
```

Comment:
id be interested to know what the team think of this for how/when to use the purple. did jen or laura provide any input regarding their thinking of when to apply purple?

Reply:
I have run it past Jen again. I think there is another pass required to resolve the use cases and scenarios, I will generate some new tickets to cover additional work.